### ISSUE TITLE
Short syntax for time estimate does not work as expected

### ISSUE THREAD

**Issue by MiragonMx** [johannesjo/super-productivity#2983]
I like the short syntax a lot and rely on it heavily during task creation. I was thrown off by the way the time estimate short syntax parsing works however.

### Your Environment

Both on PC and mobile, could also test on Linux Mint at some point.

##### Mobile: 
- Version used: 7.17.2
- Operating System and version: Android 13 on Fairphone 3+
- Build Number: 6.A.023.1
- Kernel-Version: 4.9.337-perf+
- Last Systemupdate/Securitypatch: 2023-11-01/2023-12-05

##### PC:
- Version: 7.17.2
- OS/version: Windows 10 Enterprise LTSC/21H2
- Build: 19044.3803

### Expected Behavior

From the way time estimates are displayed (e.g., 1h 30m), I would expect that such a short syntax would be supported at creation as well:  `"Test task +project 1h 30m"` for an initial estimate of one and a half hours/90 minutes (**Case A**). For (possibly(?)) easier parsing, it could also be expected, that this would only work without whitespace in the estimates' syntax, e.g. `"Test task +project 1h30m"` (**Case B**).

### Current Behavior

Currently, the short syntax only allows for one 'unit' for the estimate (day[d]/hour[h]/minute[m]), but not a combination. Everything is parsed into `xxh yym`, but similar inputs either parsed by completely neglecting the second number (**Case A**) or not parsing at all (**Case B**).

Case A:
![Case A example](https://github.com/johannesjo/super-productivity/assets/80544476/5e66473f-ed93-4232-bdcd-ec2a0889c8a8)

Case B:
![Case B example](https://github.com/johannesjo/super-productivity/assets/80544476/9433bdce-eb1c-42d4-b8da-df7ab1e1dba4)

### Steps to Reproduce (for bugs)

1. Open any Project/Today view
2. Start adding a task, either by clicking with the mouse or `Shift+A`
3. Use the described syntax to add a time estimate

### Can you reproduce this reliably?

This happens everytime. It might also be a design choice, or simply not something anyone got to yet. In the former case, I would suggest this change (or to give a hint about this in the example text), in the latter I could also create a feature request issue.

### Console Output

--

### Error Log (Desktop only)

--

**Comment by johannesjo** [comment]
As a workaround you can convert everything to minutes, e.g. 1h30m => 90m :)

But I agree that it would be nice to also be able to use `1h30m` and similiar.

**Comment by MiragonMx** [comment]
Yeah, that's what I do currently, and it's fine, I just was a bit thrown off by it and thought I'd track it here :) maybe I'll even find the time to look into it though I have very little experience in "real world" webdev projects so I expect the hurdle of entry for such a (nice but by now quite extensive and profound) project to be rather high..

**Comment by MiragonMx** [comment]
For completeness, this is still open. I'm currently trying to find some time to look at the project, so I might be able to have a go at this :)

**Comment by MiragonMx** [comment]
Bump 
Sorry, this sort of fell behind in priorities, I'll try to still make some times soon

**Comment by MiragonMx** [comment]
> Bump Sorry, this sort of fell behind in priorities, I'll try to still make some times soon

Sadly same once again 😬

**Comment by leafybees** [comment]
This appears to be closed by https://github.com/johannesjo/super-productivity/pull/5441

**Comment by MiragonMx** [comment]
Looks good! 😊
